mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -0,0 +1,38 @@
+import csv
+import io
+from datetime import datetime
+
+from moj_auth.backends import api_client
+
+from .exceptions import EmptyFileError
+
+OUTPUT_FILENAME = 'mtp_accesspay_%s.csv'
+
+
+def generate_refund_file(request):
+    client = api_client.get_connection(request)
+    refund_transactions = client.bank_admin.transactions.get(status='refund_pending')
+
+    with io.StringIO() as out:
+        writer = csv.writer(out)
+
+        refunded_transactions = []
+        for transaction in refund_transactions:
+            writer.writerow([
+                transaction['sender_sort_code'],
+                transaction['sender_account_number'],
+                transaction['sender_name'],
+                transaction['amount'],
+                transaction['reference']
```

Comment:
There was [some discussion](https://www.pivotaltracker.com/story/show/99035474) about the reference for refunds a while back. @tonyduarte: did we ever come to a conclusion on what the reference should be?